### ISSUE TITLE
Radiating pain

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9933]
**Preferred term label:**
Radiating pain

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Piriformis syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000684
- label: Radiating pain
- parents: [Pain characteristic (HP:0025280)](https://hpo.jax.org/app/browse/term/HP:0025280)
- pmid: 
- def: Pain that is perceived to extend or spread out from one area of the body to another.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9933
- synonyms:
